esmvaltool/diag_scripts/shared/statistics.ncl

```diff
@@ -1155,7 +1155,8 @@ undef("interannual_variability")
 function interannual_variability(field: numeric,
                                  y1[1]: integer,
                                  y2[1]: integer,
-                                 opt[1]: string)
+                                 opt[1]: string,
+                                 dtr[1]: string)
```

Comment:
This is a shared function, if you change it you need to make sure that other diagnostics using it are adapted (`perfmetrics/latlon.ncl` and `perfmetrics/zonal.ncl`).